cli/assets/ContractTemplate.s.sol

```diff
@@ -3,13 +3,10 @@ pragma solidity ^0.8.13;
 
 import "forge-std/Script.sol";
 
-import {Contract} from "src/Contract.sol";
-
 contract ContractScript is Script {
     function setUp() public {}
 
     function run() public {
-        vm.broadcast();
-        new Contract();
+        vm.startBroadcast();
```

Comment:
> Opinionated: Use `startBroadcast` instead of `broadcast`, as it seems to be used more often.

Heh, personally I'm a `broadcast` maxi and never use `startBroadcast`. I'm not a fan of the "filter out staticcalls" magic and prefer deploy scripts to be maximally safe and readable, which means being explicit about exactly the set of transactions you want to broadcast. You can imagine improperly defining e.g. your ERC20 interface so `balanceOf` is not a view method, then you end up accidentally sending a transaction.

If I had some really big script maybe I'd use `startBroadcast`, but for most scripts with <10 txs or something I prefer `broadcast`

Reply:
Agreed. I also thought someone could shoot themselves in the foot, but went with it anyway 🤦.
Let's change.